plonk-core/src/proof_system/preprocess.rs

```diff
@@ -297,6 +286,14 @@ where
             adicity:
                 <<F as FftField>::FftParams as ark_ff::FftParameters>::TWO_ADICITY,
         })?;
+
+        let preprocessed_table = PreprocessedLookupTable::<F, PC>::preprocess(
+            &self.lookup_table,
+            commit_key,
+            domain.size().try_into().unwrap(),
```

Comment:
We can replace with simpler:
``` rust
domain.size() as u32,
```
As a side note, I believe we are using `u32` and `u64` for n and circuit size. It would be good to use only on type throughout all the code.